features.md

```diff
@@ -160,3 +160,78 @@ Known limitations
 > editors, e.g. VSCode, provide a way to preview the rename changes before
 > applying them; while some just apply the changes directly.
 {:.tip}
+
+## Semantic Highlighting
+
+clangd computes semantic highlight tokens to enhance the source code theming. {.v9}
+
+![screenshot: highlight](screenshots/semantic_highlight.png)
+
+Tokens may be associated with one highlight kind and multiple highlight modifiers, as defined by the following section.
+
+### Kinds
+
+clangd supports following standard LSP token kinds:
+
+| Kind              | LSP Name      | Since Version | Note                       
```

Comment:
Maybe we should add an explanation here, like:

> `Kind` is the name recognized in the [config file](config#semantic-tokens), `LSP Name` is the name sent to the client.

(And a similar line for modifiers)

Reply:
I added the explanation to the config page.